packages/rx_bloc_cli/example/README.md

```diff
@@ -581,7 +581,7 @@ On iOS, an issue may occur during the initial build of the application. This can
 [patrol_pub_lnk]: https://pub.dev/packages/patrol
 [patrol_cli_pub_lnk]: https://pub.dev/packages/patrol_cli
 [patrol_native_integration_lnk]: https://patrol.leancode.pl/getting-started#integrate-with-native-side
-[patrol_integration_test_lnk]: docs/patrol_integration_test.md
-[rx_bloc_cli_cd_setup_lnk]: docs/continuous_delivery.md
-[golden_tests_with_smart_widgets_lnk]: docs/golden_tests.md
-[rx_bloc_cli_mfa_lnk]: docs/mfa.md
\ No newline at end of file
+[patrol_integration_test_lnk]: https://github.com/Prime-Holding/rx_bloc/blob/master/packages/rx_bloc_cli/example/docs/patrol_integration_test.md
```

Comment:
Why we need to use the path (full URL) here ?

Reply:
We've been doing this for some time using [this script](https://github.com/Prime-Holding/rx_bloc/blob/develop/packages/rx_bloc_cli/bin/compile_test_project.sh). I can't remember what was the reason, but this readme file is available on [pub.dev](https://pub.dev/packages/rx_bloc_cli/example).